test/api/v3/unit/libs/stripePayments.test.js

```diff
@@ -702,6 +703,7 @@ describe('Stripe Payments', () => {
           groupId,
           nextBill: currentPeriodEndTimeStamp * 1000, // timestamp in seconds
           paymentMethod: 'Stripe',
+          cancellationReason: undefined,
```

Comment:
I'm pretty sure you don't need to define cancellation reason when it is undefined.